docs/development/windows_support.md

```diff
@@ -112,12 +112,11 @@ These instructions mostly mirror the instructions in the root
   [Windows Terminal](https://learn.microsoft.com/en-us/windows/terminal/)
   and [Cmder](https://cmder.app/).
 
-#### Install tools
+#### Install tools with package manager
+
+The [choco package manager](https://chocolatey.org/install) from Chocolatey provides an easy method to install TheRock build requirements.
+Below is the commands that will install the tools required by TheRock build.
```

Comment:
At a minimum fix the "Below is the commands" grammar (mismatched singular/plural).

Suggested rewording:

```suggestion
The [choco package manager](https://chocolatey.org/install) from Chocolatey provides an easy method to install TheRock build requirements.
These commands can be used to install the tools required for building:
```